internal/app/machined/pkg/controllers/network/route_spec.go

```diff
@@ -254,6 +256,9 @@ func (ctrl *RouteSpecController) syncRoute(ctx context.Context, r controller.Run
 				OutIface: linkIndex,
 				Priority: route.TypedSpec().Priority,
 				Table:    uint32(route.TypedSpec().Table),
+				Metrics: &rtnetlink.RouteMetrics{
```

Comment:
should we send 0 MTU or should we skip it completely? I haven't looked into the kernel code to verify, but feels like it's an optional message, so we could omit it completely if MTU == 0 ?

Reply:
also it might matter when going from MTU set to some value to the case when MTU should be unset (0).